Octokit.Tests.Integration/Clients/SearchClientTests.cs

```diff
@@ -126,4 +126,392 @@ public SearchClientTests()
         Assert.NotEmpty(closedIssues);
         Assert.NotEmpty(openedIssues);
     }
+
+    [Fact]
```

Comment:
I think these should all be `[IntegrationTest]` to ensure they only run when you have an integration test account setup.